crates/bench-api/src/lib.rs

```diff
@@ -211,9 +211,22 @@ impl BenchState {
 
         let mut linker = Linker::new(&self.store);
 
-        // Import a very restricted WASI environment.
+        // Create a WASI environment.
+
         let mut cx = WasiCtxBuilder::new();
         cx.inherit_stdio();
+        // Allow access to the current working directory so that the benchmark
+        // can read its input workload(s).
+        cx.preopened_dir(
+            File::open(".").context("failed to open the current working directory")?,
+            ".",
+        );
+        // Pass this env var along so that the benchmark program can use smaller
+        // input workload(s) if it has them and that has been requested.
+        if let Ok(val) = std::env::var("WASM_BENCH_USE_SMALL_WORKLOAD") {
+            cx.env("WASM_BENCH_USE_SMALL_WORKLOAD", val);
```

Comment:
Is there a way to pass along `WASM_BENCH_*`?

Reply:
I think it makes sense to only pass through explicitly allowed env vars just so that there are less moving parts and fewer unknowns / potential ways to bias our measurements.